<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Libro</class>
 <widget class="QWidget" name="Libro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="eliminarLibro">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modificarLibro">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="agregarLibro">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabla">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <property name="autoScrollMargin">
    <number>10</number>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <column>
    <property name="text">
     <string>Titulo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Autor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Editorial</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>AÃ±o</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stock</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
